Xamarin.Forms.Core/Device.cs

```diff
@@ -58,7 +58,7 @@ public static double GetNamedSize(NamedSize size, Type targetElementType)
 			return GetNamedSize(size, targetElementType, false);
 		}
 
-		public static void OnPlatform(Action iOS = null, Action Android = null, Action WinPhone = null, Action Default = null)
+		public static void OnPlatform(Action iOS = null, Action Android = null, Action WinPhone = null, Action Default = null, Action Tizen = null)
```

Comment:
My idea is that we need to figure a better way to express this conditions, a new API that will not have to change overtime there's a new backend.. 
Also maybe that new Api could be something that combines OnPlatform with other Funds like Idiom etc.. Something more embracing.